packages/ckeditor5-ui/src/focuscycler.ts

```diff
@@ -237,6 +237,9 @@ export default class FocusCycler extends EmitterMixin() {
 		const next = this.next;
 
 		if ( next && this.focusables.getIndex( next ) === this.current ) {
+			// TBD
+			this.fire<FocusCyclerForwardCycleEvent>( 'forwardCycle' );
```

Comment:
I don't like this but this is necessary for views with just a single focusable child in their focus cycler to emit the right event and let the outside world know that they would cycle back the same focusable child (they won't, see `return` below) and maybe some parent wants to take over from here. I don't have a better idea, though.